### ISSUE TITLE
Failure in escape sequence tooltips vs complete DCC transfers notifications.

### ISSUE THREAD

**Issue by un1versal** [kvirc/KVIrc#1989]
As far as I know this has behaved like this probably always, havent tested against 4.2.0 only 4.3.1 and newer.

In theme settings I have enabled (Show tooltips for escape sequences) **boolEnableEscapeLinkTooltip**
In KVIrc DCC::File Transfer Ive enabled (Notify completion in completion in console) **boolNotifyDccSendSuccessInConsole**

You get a DCC from anyone with **any file type** console notification as follows.

```
[05:49:15]  Processing DCC SEND request from Andrio [Andrio@gateway/shell/yourbnc/x-tbuncnmgyvzugtrs] (DCC SEND Mario_Kart_Wii_by_Rosalina34.jpg 2160558192 4990 129529)
[05:49:15]  Auto-accepting DCC RECV request from Andrio!Andrio@gateway/shell/yourbnc/x-tbuncnmgyvzugtrs for file Mario_Kart_Wii_by_Rosalina34.jpg
[05:49:15]  Auto-saving DCC RECV file Mario_Kart_Wii_by_Rosalina34.jpg as /home/un1versal/Desktop/Sharelinux/incoming/Mario_Kart_Wii_by_Rosalina34.jpg
[05:49:15]  Loaded module 'filetransferwindow' (libkvifiletransferwindow.so)
[05:49:18]  DCC RECV transfer with Andrio@128.199.128.112:4990 completed: ��$
```
### Expected behavior

From tooltip contents I would expect file to be opened.with correct handler when clicking it
### Actual behavior

Upon clicking you instead get get

**source file reference:** https://github.com/kvirc/KVIrc/blob/master/src/kvirc/ui/KviIrcView_events.cpp#L1123
**media types are as follows:** https://github.com/kvirc/KVIrc/blob/cbb974beb2020ee33c0889470548b50f1c935c1c/src/kvilib/ext/KviMediaManager.cpp#L379-L393
#### **Bug 1**: ~~The tooltip is shown at ��$~~

**fixed by https://github.com/kvirc/KVIrc/commit/44e96ba435bca178892a1a831d2abe9e816f13f7**
#### **Bug 2:**  ~~warning~~

**fixed by https://github.com/kvirc/KVIrc/commit/44e96ba435bca178892a1a831d2abe9e816f13f7**
~~[05:50:07]  [KVS] Warning: Sound file '��$' not found
[05:50:07]  [KVS]   In script context "kvirc::corecall(run)", line 1, near character 1
[05:50:07]  [KVS] Warning: Sound file '��$' not found
[05:50:07]  [KVS]   In script context "kvirc::corecall(run)", line 1, near character 1
[05:50:07]  [KVS] Code listing:
[05:50:07]  [KVS]   1 play $0
[05:50:07]  [KVS] Window:
[05:50:07]  [KVS]   CONSOLE [id: 1]
[05:50:07]  [KVS] Call stack:
[05:50:07]  [KVS]   #0 Core Simple Command "play" [line 1, near character 1]
[05:50:07]  [KVS]~~
### **Bug 4:** Clicking has no longer any effect after https://github.com/kvirc/KVIrc/commit/44e96ba435bca178892a1a831d2abe9e816f13f7

> ### Edit
> 
> **bug in hardcoding **play** and unparsed file name variable in: https://github.com/kvirc/KVIrc/blob/44e96ba435bca178892a1a831d2abe9e816f13f7/src/modules/dcc/DccFileTransfer.cpp#L2203
> See comment below https://github.com/kvirc/KVIrc/issues/1989#issuecomment-223894477
### Steps to reproduce the issue
1. `/option boolEnableEscapeLinkTooltip 1`
2. `/option boolNotifyDccSendSuccessInConsole 1`
3. On any DCC complete click link like in screenshot below
   ![capture](https://cloud.githubusercontent.com/assets/3521959/15565214/c0abc97c-2310-11e6-93e3-bae2b270ea0e.PNG)
### System information

```
KVIrc 4.9.2 'Aria'

Runtime Info:
System name: Linux 3.13.0-86-generic
System version: #131-Ubuntu SMP Thu May 12 23:33:13 UTC 2016
Architecture: x86_64
Qt version: 5.5.1
Qt theme: fusion

Build Info:
Build date: 2016-05-25 14:25:25 UTC
Sources date: 20160525
Revision number: git-6681-ge7b31b8-dirty
System name: Linux-3.13.0-86-generic
CPU name: x86_64
Build command: /usr/bin/cmake
Build flags: 
   MANDIR=share/man
   CMAKE_INSTALL_PREFIX=/usr
   LIB_SUFFIX=/x86_64-linux-gnu
   Threads=POSIX
Compiler name: /usr/bin/c++
Compiler flags: N/A
Qt version: 5.5.1
Features: IRC, Debug, Crypt, SSL, IfAddr, IPC, OSS, Transparency, Webkit, Perl, Python, Enchant, Qt5, KVS
```

@TheReign ping would you have any ideas about this?

**Comment by wodim** [comment]
I fixed the display of the filename with 44e96ba but the "play" thing is hardcoded and I cba

**Comment by un1versal** [comment]
> I fixed the display of the filename with 44e96ba but the "play" thing is hardcoded and I cba

After https://github.com/kvirc/KVIrc/commit/44e96ba435bca178892a1a831d2abe9e816f13f7 theres no **warning** anymore, but clicking or double clicking does _nothing_ at all.

**Comment by un1versal** [comment]
with https://github.com/kvirc/KVIrc/commit/555a432523cad7d62ccdf5e6b6e2c94afe52f402 also fixed a possible issue with initial action when first accepting and initial saving file

> ##### Other issues:
> 
> The hardcoded variable **$0** is not parsing any argument either and also hardcoding **play** is also 100% wrong things to do.

The right things to do is determine what the correct tooltip and action would be from https://github.com/kvirc/KVIrc/blob/master/src/kvirc/ui/KviIrcView_events.cpp#L1123-L1151
